### ISSUE TITLE
[1.000] Missing Unicode Block Elements

### ISSUE THREAD

**Issue by frereit** [githubnext/monaspace#91]
It looks like [Block Elements](https://en.wikipedia.org/wiki/Block_Elements) are entirely unsupported and actually seem to break the font.

Compare the size of the three character string `a.a.a` with the size of the three character string `▒.▒.▒`:
![](https://github.com/githubnext/monaspace/assets/18506129/4d00031e-aec2-4cc3-b82f-9f8a1afc4551)

Here's what it is supposed to look like:

```
a.a.a
▒.▒.▒
```

**Comment by Morpheus2077** [comment]
Yes, missing `Box Drawing`(U+2500-257F) and `Block Elements`(U+2580-259F)

**Comment by idan** [comment]
We have added box drawing glyphs in 1.2.

Thank you!
